code/drasil-example/swhs/lib/Drasil/SWHS/TMods.hs

```diff
@@ -27,7 +27,7 @@ import Data.Drasil.Quantities.Thermodynamics (boilPt, heatCapSpec,
 import Drasil.SWHS.Assumptions (assumpHTCC, assumpTEO)
 import Drasil.SWHS.Concepts (transient)
 import Drasil.SWHS.DataDefs (ddMeltFrac)
-import Drasil.SWHS.References (incroperaEtAl2007)
+import Drasil.SWHS.References
```

Comment:
This is moving in the wrong direction. Explicit imports are much better than imports that may, randomly in the future, cause conflicts in the namespace.